frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar.dart

```diff
@@ -33,11 +35,19 @@ class HomeSideBar extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return BlocProvider(
-      create: (_) => MenuBloc(
-        user: user,
-        workspace: workspaceSetting.workspace,
-      )..add(const MenuEvent.initial()),
+    return MultiBlocProvider(
+      providers: [
+        BlocProvider(
+          create: (_) => MenuBloc(
+            user: user,
+            workspace: workspaceSetting.workspace,
+          )..add(const MenuEvent.initial()),
+        ),
+        BlocProvider(
+          create: (_) =>
+              getIt<FavoriteBloc>()..add(const FavoriteEvent.initial()),
```

Comment:
```suggestion
          create: (_) => FavoriteBloc()..add(const FavoriteEvent.initial()),
```

No need to use GetIt here.

Reply:
don't commit, just for recording.